README.md

```diff
@@ -29,11 +29,11 @@ PeekingDuck is an open-source, modular framework in Python, built for Computer V
     > peekingduck init
     ```
     The following files and folders will be created upon running `peekingduck init`:
-    - `run_config.yml` is the main configuration file for PeekingDuck. It currently contains the [default configuration](run_config.yml), and we'll show you how to modify it in a [later section](#changing-nodes-and-settings).
+    - `pipeline.yml` is the main configuration file for PeekingDuck. It currently contains the [default configuration](pipeline.yml), and we'll show you how to modify it in a [later section](#changing-nodes-and-settings).
```

Comment:
Should we change the explanation from configuration to pipeline as well?

Reply:
The terminology is one that I'm not 100% decided, 'coz we are dealing with "pipeline configuration", so both "pipeline" and "configuration" seem admissible...